### DISCUSSION TITLE
[Miscellaneous] Bridge Collab.land token from ETH mainnet  to Optimism

### DISCUSSION THREAD

**Discussion by DevvyHall** [ethereum-optimism/developers#501] [Miscellaneous]
### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

I am trying to bridge a token directly from the smart contract (0x99C9fc46f92E8a1c0deC1b1747d010903E884bE1) using the etherscan interface. I have tried using the depositERC20 function, filling in the following data:
_l1Token address 0x8B21e9b7dAF2c4325bf3D18c1BeB79A347fE902A
_l2Token address 0x8B21e9b7dAF2c4325bf3D18c1BeB79A347fE902A
_amount   147872000000000000000000
_minGasLimit  200000
_extraData   0x
I execute the transaction and it remains pending. I don't understand why this happens to me if a similar transaction has had no problems bridging before (tx: 0xd0a7233c9175619bda09eddadcdda4fce9815b2a7af2e536df8f0b6eeebbf8e9)
![SC111](https://github.com/ethereum-optimism/developers/assets/138037296/834d0e46-9f39-4414-bd8e-b81d8bc2a130)

**Comment by hash1go** [comment]
I just want to confirm, has the Approve transaction been submitted?

**Comment by imtipi** [comment]
if you provide your address,i can run this tx in tenderly for debugging
